chitterApi.js

```diff
@@ -0,0 +1,62 @@
+class ChitterApi {
+  fetchPeeps(callback) {
+    fetch("https://chitter-backend-api-v2.herokuapp.com/peeps")
+      .then((response) => response.json())
+      .then(callback);
+  }
+
+  createUser(newUsername, newPassword, callback) {
+    fetch("https://chitter-backend-api-v2.herokuapp.com/users", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        user: { handle: newUsername, password: newPassword },
+      }),
+    })
+      .then((response) => response.json())
+      .then(callback);
+  }
+
+  loginUser(username, password, callback) {
+    fetch("https://chitter-backend-api-v2.herokuapp.com/sessions", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        session: { handle: username, password: password },
+      }),
+    })
+      .then((response) => response.json())
+      .then(callback);
+  }
+
+  postPeep(sessionKey, userId, peepBody, callback) {
+    fetch("https://chitter-backend-api-v2.herokuapp.com/peeps", {
+      method: "POST",
+      headers: {
+        Authorization: `Token token=${sessionKey}`,
```

Comment:
Interesting difference here. 

Will also work as 'Token token=' + sessionKey

I don't know which is best.